### ISSUE TITLE
Javascript Support

### ISSUE THREAD

**Issue by NelsonBrandao** [simurai/duotone-syntax#14]
Hi,

I love this theme, but i think the JS language support is a bit lacking. For objects the colors are all the same.

![JS Example](http://i.imgur.com/MBzP1BV.png)

We could do something similar to SCSS with different color for the keys, and maybe differentiate between `bool`, `integer` and `string`

![CSS Example](http://i.imgur.com/QQcos3K.png)

What do you guys think?

**Comment by simurai** [comment]
Yeah, that's a good idea. Left a bit toned down and then right stronger.

**Comment by simurai** [comment]
Hmm.. @NelsonBrandao I just tried your example, and it seems already close to how SCSS does it:

![screen shot 2016-04-17 at 5 38 15 pm](https://cloud.githubusercontent.com/assets/378023/14585960/48e5f0b8-04c3-11e6-9fa6-4b3822e8f8cc.png)

Do you use a different JS language package?

**Comment by NelsonBrandao** [comment]
Hmm, yup you are right. This only happens with `Babel ES6 Javascript`

**Comment by simurai** [comment]
What's language package you use? Maybe we can add better support in this theme. Or maybe that package needs to be changed.
